### ISSUE TITLE
Disable branch out of date check

### ISSUE THREAD

**Issue by max-wittig** [python-gitlab/python-gitlab#604]
This hinders MR from being merged and created ugly `Merged master into 'branch'` commits, when people don't know how to use `git rebase`.

If the branch has conflicts, GitHub will prevent merging anyway.

**Comment by gpocentek** [comment]
Agreed! I've changed the setting. Let me know if something else is required to make PR easier to review/merge.
